website/pages/docs/plugin-portal/index.mdx

```diff
@@ -101,6 +101,8 @@ reviewed by HashiCorp.
 
 - [Venafi](https://github.com/Venafi/vault-pki-backend-venafi)
 
+- [PrimeKey EJBCA](https://github.com/primekeydevs/ejbca-vault-plugin)
+
```

Comment:
Changing the order since the list should be alphabetical.

```suggestion
- [PrimeKey EJBCA](https://github.com/primekeydevs/ejbca-vault-plugin)
- [Venafi](https://github.com/Venafi/vault-pki-backend-venafi)
```